Assets/Scripts/Input/AetherControlSystem.cs

```diff
@@ -845,6 +883,8 @@ public @AetherControlSystem()
         m_Player_Fire = m_Player.FindAction("Fire", throwIfNotFound: true);
         m_Player_Jump = m_Player.FindAction("Jump", throwIfNotFound: true);
         m_Player_SetBomb = m_Player.FindAction("SetBomb", throwIfNotFound: true);
+        m_Player_LaunchMeteor = m_Player.FindAction("LaunchMeteor", throwIfNotFound: true);
+        m_Player_LaunchForcefield = m_Player.FindAction("LaunchForcefield", throwIfNotFound: true);
```

Comment:
Might want to take out the specific input stuff, since it probably won't be called using this when we activate the skills.